src/content/docs/dev/amplifier/gmp-example.mdx

```diff
@@ -6,127 +6,141 @@ This example shows the execution of a GMP call from the Ethereum Sepolia to the
 
 You will need:
 
-- An [`axelard` node](/dev/amplifier/chain-integration/introduction/) that is [connected to the devnet](/validator/amplifier/verifier-onboarding/)
+- A [funded wallet](https://docs.axelar.dev/dev/amplifier/chain-integration/integrate-a-chain/#make-sure-you-have-a-funded-wallet) on `devnet-amplifier`
 - A [Solidity contract](https://github.com/cjcobb23/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol) deployed to the Ethereum Sepolia and Avalanche Fuji testnets
+  - We use a [sample contract](https://gist.github.com/StephenFluin/9379a9fc202d54854fd74f42bcac3178) throughout this demo
```

Comment:
Any reason we're linking this contract rather than the one from the axelar [examples repo](https://github.com/axelarnetwork/axelar-examples/blob/main/examples/evm/call-contract/CallContract.sol)? (also a simple gmp msg)

Reply:
Wasn't aware we had that 😅, I've updated the link. The contract from the previous link isn't *exactly* the same but I think it should be fine